pkg/process/process_elasticsearch.go

```diff
@@ -38,9 +39,12 @@ func (p *ElasticSearch) StartLocal(logfile string, opts ...StartOp) error {
 
 func (p *ElasticSearch) StartElasticSearch(logfile string, opts ...StartOp) error {
 	// simple single node cluster
+	if p.Port == "" {
```

Comment:
We don't run anything else on 9200. Is this something specific to your setup, or do we not clean something up properly?

Reply:
So I was (and still am) seeing intermittent failures in the TestEvents unit test. It always passes when I run it standalone, but it fails in `make unit-test`. I suspected that multiple unit tests are trying to write events. Normally these just fail silently, but if they overlap with the TestEvents test, which is actually checking the state of events, it might add extra junk which causes the test to fail. That was my hypothesis anyway, so I though if I changed the port for the TestEvents test, the other tests wouldn't be able to write to ES. Unfortunately i think I still see that test failing sometimes.

Reply:
hm, ok. Yeah, since multiple unit-tests will run in parallel its hard to figure out how to properly address this, but if this works, that's fine.